src/redisraft.c

```diff
@@ -646,9 +646,32 @@ static void handleRedisCommandAppend(RedisRaftCtx *rr,
         return;
     }
 
-    if (cmd_flags & CMD_SPEC_DENYOOM && (RedisModule_GetUsedMemoryRatio() > 1.0)) {
-        RedisModule_ReplyWithError(ctx, "OOM command not allowed when used memory > 'maxmemory'.");
-        return;
+    /* dry run */
+    for (int i = 0; i < cmds->len; i++) {
+        RaftRedisCommand *cmd = cmds->commands[i];
+        size_t cmdlen;
+        const char *cmdstr = RedisModule_StringPtrLen(cmd->argv[0], &cmdlen);
+        /* skip multi */
+        if (i == 0 && cmdlen == 5 && !strncasecmp(cmdstr, "MULTI", 5)) {
```

Comment:
Why do we skip multi? If I'm not mistaken, ACL can reject it? Even it won't reject it, do we need this special check?